### PR TITLE
feat(callbar-button): DLT-2211 add passthrough tooltip props

### PR DESCRIPTION
**Pull request by braddialpad**
# feat(callbar-button): add passthrough tooltip props

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will increment the version number on release:

- [x] Feature

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-2211

## :book: Description

Added a few requested tooltip props to callbar_button and callbar_button_with_popover

## :bulb: Context

Needed to handle some edge cases in DPM

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.
- [x] I have added all relevant documentation.
- [x] I have considered the performance impact of my change.

For all Vue changes:

- [x] I have made my changes in Vue 2 and Vue 3. Note: you may sync your changes from Vue 2 to Vue 3 (or vice versa) using the `./scripts/dialtone-vue-sync.sh` script. Read docs here: [Dialtone Vue Sync Script](../packages/dialtone-vue3/.github/CONTRIBUTING.md#dialtone-vue-sync-script)

### CODE REVIEW

File: packages/dialtone-vue2/recipes/buttons/callbar_button/callbar_button_default.story.vue [hunk 1]
```diff
@@ -10,6 +10,9 @@
     :button-class="$attrs.buttonClass"
     :text-class="$attrs.textClass"
     :button-width-size="$attrs.buttonWidthSize"
+    :inverted="$attrs.inverted"
+    :show="$attrs.show"
+    :tooltip-text="$attrs.tooltipText"
```
Comment (juliodialpad) [inline packages/dialtone-vue2/recipes/buttons/callbar_button/callbar_button_default.story.vue]:
```suggestion
    :inverted-tooltip="$attrs.invertedTooltip"
    :show-tooltip="$attrs.showTooltip"
    :tooltip-text="$attrs.tooltipText"
```

File: packages/dialtone-vue2/recipes/buttons/callbar_button_with_popover/callbar_button_with_popover_default.story.vue [hunk 2]
```diff
@@ -16,6 +16,9 @@
     :text-class="$attrs.textClass"
     :content-class="$attrs.contentClass"
     :open-popover="$attrs.openPopover"
+    :inverted="$attrs.inverted"
+    :show="$attrs.show"
+    :tooltip-text="$attrs.tooltipText"
```
Comment (juliodialpad) [inline packages/dialtone-vue2/recipes/buttons/callbar_button_with_popover/callbar_button_with_popover_default.story.vue]:
```suggestion
    :inverted-tooltip="$attrs.invertedTooltip"
    :show-tooltip="$attrs.showTooltip"
    :tooltip-text="$attrs.tooltipText"
```

File: packages/dialtone-vue3/recipes/buttons/callbar_button/callbar_button_default.story.vue [hunk 3]
```diff
@@ -10,6 +10,9 @@
     :button-class="$attrs.buttonClass"
     :text-class="$attrs.textClass"
     :button-width-size="$attrs.buttonWidthSize"
+    :inverted="$attrs.inverted"
+    :show="$attrs.show"
+    :tooltip-text="$attrs.tooltipText"
```
Comment (juliodialpad) [inline packages/dialtone-vue3/recipes/buttons/callbar_button/callbar_button_default.story.vue]:
```suggestion
    :inverted-tooltip="$attrs.invertedTooltip"
    :show-tooltip="$attrs.showTooltip"
    :tooltip-text="$attrs.tooltipText"
```

File: packages/dialtone-vue3/recipes/buttons/callbar_button_with_popover/callbar_button_with_popover_default.story.vue [hunk 4]
```diff
@@ -16,6 +16,9 @@
     :text-class="$attrs.textClass"
     :content-class="$attrs.contentClass"
     :open-popover="$attrs.openPopover"
+    :inverted="$attrs.inverted"
+    :show="$attrs.show"
+    :tooltip-text="$attrs.tooltipText"
```
Comment (juliodialpad) [inline packages/dialtone-vue3/recipes/buttons/callbar_button_with_popover/callbar_button_with_popover_default.story.vue]:
```suggestion
    :inverted-tooltip="$attrs.invertedTooltip"
    :show-tooltip="$attrs.showTooltip"
    :tooltip-text="$attrs.tooltipText"
```

**Review by juliodialpad — CHANGES_REQUESTED** [review]
Apply the requested changes and make sure all the properties work on both vue 2 and vue 3 storybook's

**Comment by braddialpad** [comment]
Changes made, nice find thanks!

**Comment by braddialpad** [comment]
Accessibility tests failing due to some installation issue, nothing to do with the PR

**Review by juliodialpad — CHANGES_REQUESTED** [review]
Please add `invertedTooltip` and `showTooltip` props to `callbar_button.stories.js` as on Vue 2 storybook doesn't render those correctly and the props don't work as expected.

**Comment by braddialpad** [comment]
> Please add `invertedTooltip` and `showTooltip` props to `callbar_button.stories.js` as on Vue 2 storybook doesn't render those correctly and the props don't work as expected.

Weird.. wonder why that only happens in vue 2.

**Review by juliodialpad — APPROVED** [review]
Working on `callbar_button_with_popover.stories.js` just add it to `callbar_button.stories.js` before merging and this should be good to go, thanks!

**Pull request merged**